wdl/deprecated/tasks/Pacasus.wdl

```diff
@@ -137,8 +137,8 @@ task RemovePalindromes {
 
     runtime {
         cpu:                    default_attr.cpu_cores
-        memory:                 default_attr.mem_gb + " GiB"
-        disks: "local-disk " +  default_attr.disk_gb + " HDD"
+        memory:                 select_first([default_attr.mem_gb]) + " GiB"
```

Comment:
I updated this so that `miniwdl check` would pass.  I was running it on all wdls in the repo before I remembered that the `deprecated` folder was not being checked.  It's not being used anywhere.

Reply:
I follow the same logic as Jonn.
That is, I validate locally too, including the `deprecate`d ones. So if any of my changes makes the deprecated ones invalid, I'd do the minimal to make them valid but nothing more.
That being said, I don't fix bugs in those deprecated pipelines or tasks.